### ISSUE TITLE
Relative (anchor) links in readmes

### ISSUE THREAD

**Issue by FichteFoll** [wbond/packagecontrol.io#27]
Example: 

https://sublime.wbond.net/packages/InsertDate Usage > Screenshot > "Settings"

This is supposed to be a relative link to the settings section. The renderer, however, does not add the anchors from headers to the source and relativates the link to `https://raw.githubusercontent.com/FichteFoll/sublimetext-insertdate/master/#settings`, which is just wrong.

**Comment by FichteFoll** [comment]
This has been improved with 72f39e3420d3cecedf79fccb4cc407487ecc387e (the links) but the anchors don't exist yet since the used MD converter doesn't create them.

**Comment by wbond** [comment]
Yeah, I wish GitHub still maintained their markdown converter as an open source project

**Comment by teddybeermaniac** [comment]
Hi, I encountered this bug today and after some research I found out that:

For markdown, [Markdown](https://pythonhosted.org/Markdown/index.html) library supports this via its TOC extension:

    >>> markdown.markdown('[Settings](#settings)\n### Settings ###', extensions=['markdown.extensions.toc'])
    '<p><a href="#settings">Settings</a></p>\n<h3 id="settings">Settings</h3>'

In case of reStructuredText, [ReText](http://sourceforge.net/projects/retext) outputs HTML with working links and it uses [Markups](https://pythonhosted.org/Markups) module for conversion:

    >>> markups.ReStructuredTextMarkup().get_document_body('`Settings`_\n\n========\nSettings\n========\n')
    '<p><a class="reference internal" href="#settings">Settings</a></p>\n<div class="section" id="settings">\n<h1>Settings</h1>\n</div>\n'

**Comment by FichteFoll** [comment]
:+1:
